### PR TITLE
DSD-646: Label bottom margin

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-646](https://jira.nypl.org/browse/DSD-646)

## This PR does the following:

- Updates the bottom margin for labels and legends in form components from "s" to "xs". The components affected include: `CheckboxGroup`, `DatePicker`, `Fieldset`, `RadioGroup`, `Select`, `Slider`, and `TextInput`.

## How has this been tested?

In Storybook. `Select` and `TextInput` are missing snapshot tests but I'll create a separate ticket for that.

## Accessibility concerns or updates

- None, it's just appearance.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr807-2vngdticla7tiiwhrjjbenpvcljjljys.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b7bca05f46ed13d8cdc178

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Adds export statements for `ProgressIndicator` and `Slider` components to `index.ts`.
 - Adds `Blogs` variant to `Breadcrumbs` component.
 
+### Changes
+
+- Updates the bottom margin on labels and legends for form components.
```
Comment (bigfishdesign13) [inline CHANGELOG.md]: Can this be more specific?  `Reduces the bottom margin from 16px to 8px...`

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
I didn't know this was a one-liner.  Awesome.  Just asking for one minor alteration to the CHANGELOG entry.

**Comment by EdwinGuzman** [comment]
Yeah, I'm glad these are one-liners because it makes the styles more consistent throughout.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr807-2vngdticla7tiiwhrjjbenpvcljjljys.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b7bca05f46ed13d8cdc178

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr807-2vngdticla7tiiwhrjjbenpvcljjljys.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b7bca05f46ed13d8cdc178

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  Thanks for making the change.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr807-2vngdticla7tiiwhrjjbenpvcljjljys.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b7bca05f46ed13d8cdc178

**Pull request merged**